O365WebHook/formatO365Log.js

```diff
@@ -0,0 +1,33 @@
+const Parse = require('al-collector-js').Parse;
+
+module.exports = function(item) {
+    //Paths from https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#common-schema
+    const creationTimePaths = [
+        {path: ['CreationTime']}
+    ];
+    const messageTypeIdPaths = [
+        {path: ['Operation']}
```

Comment:
`RecordType` was used as `messageTypeId` previously. Please fix it.

Reply:
So when i tried to use record type, per the previous version, the library or api threw and error saying that the MessageType should be a string. I thought it was a bad smell to stringify that integer just to satisfy the library.

Reply:
we need to fix the library then.
We have to use `RecordType` here.